src/bsoncxx/document/element.hpp

```diff
@@ -78,20 +78,6 @@ class BSONCXX_API element {
     ///
     element();
 
-    ///
```

Comment:
This constructor was unsafe to have public. Made it private and friended in the necessary things.